### ISSUE TITLE
Bug from df8c6e2: TypeError: sequence item 1: expected str instance, int found

### ISSUE THREAD

**Issue by scpeters** [philipstarkey/bitbucket-hg-exporter#42]
I've been working on a larger export using a branch on my fork (https://github.com/scpeters/bitbucket-hg-exporter/commit/aa01adf301c62584c734a9b4ea5145460a9e69b2), and there seems to be a bug in some of the code added in df8c6e204e08f896ccee0f9942f5aa36d5d9c553:

~~~
? Overwrite HTML app for GitHub pages site with latest version?  Yes                                                               
Reordering comments...
Traceback (most recent call last):
  File "/data_fast/scpeters/ws/bb_venv/bin/bitbucket-hg-exporter", line 11, in <module>
    load_entry_point('bitbucket-hg-exporter==0.7.0', 'console_scripts', 'bitbucket-hg-exporter')()
  File "/data_fast/scpeters/ws/bb_venv/lib/python3.7/site-packages/bitbucket_hg_exporter/__main__.py", line 2499, in main
    project = MigrationProject()
  File "/data_fast/scpeters/ws/bb_venv/lib/python3.7/site-packages/bitbucket_hg_exporter/__main__.py", line 281, in __init__
    self.__load_project(**kwargs)
  File "/data_fast/scpeters/ws/bb_venv/lib/python3.7/site-packages/bitbucket_hg_exporter/__main__.py", line 312, in __load_project
    self.__confirm_project_settings(load=True)
  File "/data_fast/scpeters/ws/bb_venv/lib/python3.7/site-packages/bitbucket_hg_exporter/__main__.py", line 1022, in __confirm_project_settings
    new_file_path = '='.join(new_file_path_prefixes+[page_num]) + '.' + extension
TypeError: sequence item 1: expected str instance, int found
~~~

**Comment by philipstarkey** [comment]
That should be `[str(page_num)]`

That block of code only gets hit if we need to add a dummy comment and that addition pushes the number of comments over a multiple of 100 (so a reasonably rare event!)

**Comment by scpeters** [comment]
yeah, I just tried that, and it seems to work

~~~
diff --git a/bitbucket_hg_exporter/__main__.py b/bitbucket_hg_exporter/__main__.py
index d6dd445..f5eb293 100644
--- a/bitbucket_hg_exporter/__main__.py
+++ b/bitbucket_hg_exporter/__main__.py
@@ -1019,7 +1019,7 @@ class MigrationProject(object):
                             *new_file_path_prefixes, new_file_path_suffix = comment_files[-1].split('=')
                             page_num, extension = new_file_path_suffix.split('.')
                             page_num = int(page_num) + 1
-                            new_file_path = '='.join(new_file_path_prefixes+[page_num]) + '.' + extension
+                            new_file_path = '='.join(new_file_path_prefixes+[str(page_num)]) + '.' + extension
                             # open old file and add "next" page
                             with open(comment_files[-1], 'r') as f:
                                 prev_file_data = json.load(f)
~~~

yeah, I was surprised too

**Comment by scpeters** [comment]
https://github.com/philipstarkey/bitbucket-hg-exporter/pull/43

**Comment by philipstarkey** [comment]
Fix released in 0.7.1
